packages/block-library/src/site-title/edit/index.js

```diff
@@ -26,14 +28,49 @@ export default function SiteTitleEdit( {
 	setAttributes,
 	insertBlocksAfter,
 } ) {
-	const { level, textAlign } = attributes;
+	const { level, textAlign, isLink, linkTarget } = attributes;
 	const [ title, setTitle ] = useEntityProp( 'root', 'site', 'title' );
 	const TagName = level === 0 ? 'p' : `h${ level }`;
 	const blockProps = useBlockProps( {
 		className: classnames( {
 			[ `has-text-align-${ textAlign }` ]: textAlign,
 		} ),
 	} );
+
+	let titleElement = (
```

Comment:
How can I reduce this to remove the duplication?

Reply:
Done in https://github.com/WordPress/gutenberg/pull/31540/commits/1c64ce0cd953b0f095bc92760286ec4488cd8d12 :+1: